main/main.c

```diff
@@ -2255,6 +2255,7 @@ int php_module_startup(sapi_module_struct *sf, zend_module_entry *additional_mod
 
 	zuv.html_errors = 1;
 	zuv.import_use_extension = ".php";
+	zuv.import_use_extension_length = (uint)strlen(zuv.import_use_extension);
```

Comment:
Isn't this line is better to be
zuv.import_use_extension_length = sizeof(".php") - 1;

We use sizeof("str") - 1, cases like this.